torch/_dynamo/variables/misc.py

```diff
@@ -410,38 +406,6 @@ def call_method(
                 )
             else:
                 return self.call_apply(tx, args, kwargs)
-        elif name == "backward":
-            with tx.strict_translation_mode():
-                if isinstance(self.fn_cls.backward, types.FunctionType):
-                    backward = UserFunctionVariable(self.fn_cls.backward)
-                elif isinstance(self.fn_cls.backward, types.MethodType):
-                    backward = UserMethodVariable(
-                        self.fn_cls.backward.__func__,
-                        variables.UserDefinedClassVariable(self.fn_cls),
-                    )
-                    args = [backward.obj] + args
-                else:
-                    unimplemented(
-                        f"backward is a non-function or method: {self.fn_cls.backward}"
-                    )
-
-                return tx.inline_call(tx, backward, args, kwargs)
-
-        elif name == "forward":
-            if isinstance(self.fn_cls.forward, types.FunctionType):
-                forward = UserFunctionVariable(self.fn_cls.forward)
-            elif isinstance(self.fn_cls.forward, types.MethodType):
-                forward = UserMethodVariable(
-                    self.fn_cls.forward.__func__,
-                    variables.UserDefinedClassVariable(self.fn_cls),
-                )
-                args = [forward.obj] + args
-            else:
-                unimplemented(
-                    f"forward is a non-function or method: {self.fn_cls.forward}"
-                )
-
-            return tx.inline_call(tx, forward, args, kwargs)
```

Comment:
I think these are dead code after #116897, as the ```forward/backward``` functions tracing were triggered in ```AutogradFunctionApplyVariable.call_function```.